src/Cryptol/Backend/Monad.hs

```diff
@@ -139,7 +139,7 @@ data ThunkState a
        --   a tight loop evaluating this thunk.  If the thunk is being evaluated
        --   by some other thread, the current thread will await it's completion.
```

Comment:
You missed the same typo two lines earlier :)

Reply:
c'est la vie